docs/advanced.rst

```diff
@@ -806,76 +806,168 @@ DiscoverySettings
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<partitionQos>``     | The name of a partition of the    | ``string``        |              |
+|                        | remote peer. Repeat to configure  |                   |              |
+|                        | several partitions.               |                   |              |
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<unicastLocator>``   | Unicast locator of the            |                   |              |
+|                        | participant.                      |                   |              |
+|                        | see :ref:`staticLocators`.        |                   |              |
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<multicastLocator>`` | Multicast locator of the          |                   |              |
+|                        | participant.                      |                   |              |
+|                        | see :ref:`staticLocators`.        |                   |              |
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<reliabilityQos>``   | See the :ref:`reliability`        | |besteffort|,     | |besteffort| |
+|                        | section.                          | |reliable|        |              |
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<durabilityQos>``    | See the                           | |volatile|,       | |volatile|   |
+|                        | :ref:`SettingDataDurability`      | |transientlocal|, |              |
+|                        | section.                          | |transient|       |              |
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<ownershipQos>``     | See                               |                   |              |
+|                        | :ref:`ownershipQos`.              |                   |              |
++------------------------+-----------------------------------+-------------------+--------------+
+| ``<livelinessQos>``    | Defines the liveliness of the     |                   |              |
+|                        | remote peer.                      |                   |              |
+|                        | see :ref:`livelinessQos`.         |                   |              |
++------------------------+-----------------------------------+-------------------+--------------+
+
+
+.. _staticLocators:
+
+Locator definition
+------------------
+
+Locators for remote peers are configured using ``<unicastLocator>`` and ``<multicastLocator>`` tags.
+These take no value, and the locators are defined using tag attributes:
+* :class:`address`: a mandatory ``string`` representing the locator address.
```

Comment:
A blank line is necessary before this one. Otherwise, the bullet list is not generated.
```suggestion

* :class:`address`: a mandatory ``string`` representing the locator address.
```